addons/website_portal_purchase/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Website Portal for Purchases',
+    'category': 'Website',
+    'summary': 'Add your purchase orders in the frontend portal',
+    'version': '1.0',
+    'description': """
+Add your purchase orders in the frontend portal. Your customers will be able to connect to their portal to see the list (and the state) of their purchase orders.
```

Comment:
Display purchase orders on your website. Customers will be able to access their purchase orders in their own portal, see their state and communicate with salesmen.